lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3524,6 +3525,14 @@ namespace {
             hasUnreferenceableStorage = true;
             hasMemberwiseInitializer = false;
           }
+
+          if (auto cxxdecl = dyn_cast<clang::CXXRecordDecl>(nd)) {
+            if (cxxdecl->isAnonymousStructOrUnion() && cxxdecl->isUnion()) {
```

Comment:
We can flatten this if with the one below it.

Reply:
Yeah I think we could have a broader unimportable cxx decls list and use it in different places sure.